### PR TITLE
Remove advanced manager

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Commons/pull/88

### CODE REVIEW

File: octobot_trading/api/modes.py [hunk 1]
```diff
@@ -15,18 +15,13 @@
...
 from octobot_commons.logging.logging_util import get_logger
-from octobot_commons.tentacles_management.advanced_manager import create_classes_list
 
 from octobot_trading.api import LOGGER_TAG
 from octobot_trading.exchanges.exchange_manager import ExchangeManager
 from octobot_trading.modes import AbstractTradingMode
 from octobot_trading.util.trading_config_util import get_activated_trading_mode as util_get_activated_trading_mode
 
 
-def init_trading_mode_config(config) -> None:
```
Comment (Herklos) [inline octobot_trading/api/modes.py:26]: :+1:

File: octobot_trading/data/portfolio_profitability.py [hunk 2]
```diff
@@ -125,13 +124,20 @@ async def _update_profitability(self, force_recompute_origin_portfolio=False):
...
                          in self._only_symbol_currency_filter(self.current_crypto_currencies_values).items()
                          if self.origin_crypto_currencies_values[currency] > 0]
 
         return sum(origin_values) / len(origin_values) * 100 - 100 if origin_values else 0
 
+    def _get_origin_value_using_ref_market(self, currency):
+        # preferably compute value in reference market
+        ref_market_pair = merge_currencies(currency, self.reference_market)
+        if ref_market_pair in set(self.origin_crypto_currencies_values.keys()):
+            return self.origin_crypto_currencies_values[ref_market_pair]
```
Comment (Herklos) [inline octobot_trading/data/portfolio_profitability.py:138]: I don't understand why in sometimes we use `currencies` as keys and some other times `pairs`

Comment (GuillaumeDSM) [reply octobot_trading/data/portfolio_profitability.py:138]: Right, I removed it and it also fixed an issue with initial portfolio value

Comment (Herklos) [reply octobot_trading/data/portfolio_profitability.py:138]: ok :+1:

**Pull request merged**